Source/DataSources/PositionPropertyArray.js

```diff
@@ -104,8 +104,8 @@ define([
      *
      * @param {JulianDate} time The time for which to retrieve the value.
      * @param {ReferenceFrame} referenceFrame The desired referenceFrame of the result.
-     * @param {Cartesian3} [result] The object to store the value into, if omitted, a new instance is created and returned.
-     * @returns {Cartesian3} The modified result parameter or a new instance if the result parameter was not supplied.
+     * @param {Cartesian3[]} [result] The object to store the value into, if omitted, a new instance is created and returned.
```

Comment:
Should this say "array of objects" instead of "object"?  And is it a breaking API change?

Reply:
Not a breaking change, the documentation was wrong, not the code.